btree/btree.c

```diff
@@ -3758,13 +3766,18 @@ static int64_t btree_get_tick(struct m0_sm_op *smop)
 
 		bop->bo_cb.c_act(&bop->bo_cb, &s.s_rec);
 
-		lock_op_unlock(tree);
+		lock_op_unlock(tree, oi);
 		return P_CLEANUP;
 	}
-	case P_CLEANUP:
+	case P_CLEANUP: {
 		level_cleanup(oi, bop->bo_tx);
+		return P_FINI;
```

Comment:
it should be `return m0_sm_op_ret(&bop->bo_op);`

Reply:
you want to add return m0_sm_op_ret(&bop->bo_op); at P_CLENUP phase of P_FINI phase

Reply:
done